### PR TITLE
Sowri

### PR DESCRIPTION
**Pull request by Sowri19**

### CODE REVIEW

**Review by shashibyadav — CHANGES_REQUESTED** [review]

File: src/Components/Camera/index.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  const takePicture = async () => {
+    if (cameraRef.current) {
+      const photo = await cameraRef.current.takePictureAsync();
+      onPictureTaken(photo);
+    }
+  };
+
+  return (
+    <OverlayContainer>
```
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Components/Camera/index.tsx:48]: fixed

File: src/Components/Camera/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+export const TakePictureButton = styled.TouchableOpacity`
+  position: absolute;
+  bottom: 20px;
+  align-self: center;
+  padding: 10px;
+  background-color: #2c075a;
+`;
+
+export const ButtonText = styled.Text`
```
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Components/Camera/styles.tsx:57]: fixed

File: src/Components/Camera/styles.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  color: white;
+  font-size: 20px;
+  margin-left: 10px;
+`;
+export const ButtonText1 = styled.Text`
```
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Components/Camera/styles.tsx:62]: fixed

File: src/Components/Camera/styles.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  color: #2c075a;
+  font-size: 20px;
+  margin-left: 10px;
+`;
+
+export const Text = styled.Text`
```
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Components/Camera/styles.tsx:68]: fixed

File: src/Pages/Login-Register/Register/AccountCreation/Step6/index.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      });
+
+      console.log('Image saved to:', fileUri);
+    } catch (error) {
+      console.error('Error saving the image file:', error);
+    }
+  };
+
+  return (
+    <Container>
```
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Pages/Login-Register/Register/AccountCreation/Step6/index.tsx:26]: fixed

File: src/Pages/Login-Register/Register/AccountCreation/Step6/styles.tsx [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components/native';
+
+export const Container = styled.View`
```
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Pages/Login-Register/Register/AccountCreation/Step6/styles.tsx:3]: fixed

File: src/utilities/FacialRecognitionOverlay.png
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/utilities/FacialRecognitionOverlay.png:1]: fixed

File: src/Components/Camera/styles.tsx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+export const CenterFaceButton = styled(Button)`
+  position: absolute;
+  bottom: 180px;
+  align-self: center;
+  width: 293px;
+  height: 57px;
+  background-color: white;
+`;
+
+export const Container = styled.View`
```
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Components/Camera/styles.tsx:21]: fixed

File: src/Components/Camera/index.tsx [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+const CustomCamera = ({
+  onPictureTaken,
+  initialCameraType,
+}: CustomCameraProps) => {
+  const [hasCameraPermission, setHasCameraPermission] = useState<
+    boolean | null
+  >(null);
+  const cameraRef = useRef<Camera | null>(null);
+
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Components/Camera/index.tsx:25]: fixed

File: src/Components/Camera/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sowri19) [reply src/Components/Camera/index.tsx:48]: fixed

**Pull request merged**